docs/pages/mydoc/rawapi.md

```diff
@@ -125,3 +125,25 @@ It is not possible to read and update in the same transaction.
     {ok, Result, CT2} = rpc:call(Node, antidote, read_objects, [CT1, [], [CounterObj, SetObj]]),
     [CounterVal, SetVal] = Result.
 ```
+
+## Cluster management
+
+An Antidote data center (DC) is a cluster of multiple antidote nodes. Antidote deployment can have multiple DCs where each DCs is a full replica. The cluster management API provides functions to create DCs and connect them.
+
+#### Example ####
+If 6 antidote nodes have already started, we can create two DCs with 3 nodes each as follows:
+
+```erlang
+     rpc:call('antidote@node1', create_dc, [['antidote@node1', 'antidote@node2', 'antidote@node3']]),
+     rpc:call('antidote@node4', create_dc, [['antidote@node4', 'antidote@node5', 'antidote@node6']]).
+```
+
+To connect two DCs:
+
+```erlang
+     {ok, Descriptor1} = rpc:call('antidote@node1', get_connection_descriptor, []),
+     {ok, Descriptor2} = rpc:call('antidote@node4', get_connection_descriptor, []),
+     Descriptors = [Descriptor1, Descriptor2],
+     rpc:call('antidote@node1', subscribe_updates_from, [Descriptors]),
```

Comment:
What happens if only one of the nodes subscribes for updates? Might be a typical error.
Would it make sense to have a function for this connecting functionality (taking as parameter the DC some nodes)?
Actually, can we use arbitrary nodes for connecting the corresponding DCs?